common/lib/xmodule/xmodule/js/spec/video/video_events_plugin_spec.js

```diff
@@ -78,6 +78,13 @@
                 new_time: 1,
                 type: 'any'
             });
+            expect(state.videoEventsPlugin.emitPlayVideoEvent).toBeTruthy();
+        });
+
+        it('can emit "play_video" event after "seek_video" event ', function() {
+            state.videoEventsPlugin.emitPlayVideoEvent = false;
```

Comment:
That is because `videoPlayer`'s state is destroyed after each test case at [this point](https://github.com/edx/edx-platform/blob/46797666556566bb16d05e3914087c274323aaa6/common/lib/xmodule/xmodule/js/spec/video/video_events_plugin_spec.js#L22). And when it is initialized, it do it with `[this.emitPlayVideoEvent = true;](https://github.com/edx/edx-platform/blob/46797666556566bb16d05e3914087c274323aaa6/common/lib/xmodule/xmodule/js/src/video/09_events_plugin.js#L57-L57)`.